deploy/__main__.py

```diff
@@ -27,7 +27,9 @@
     user_data=user_data,
     ssh_keys=[ssh_key.fingerprint])
 
-public_ip = do.ReservedIp("cobra-public-ip", droplet_id=droplet.id, region=droplet.region)
+public_ip = do.ReservedIp("cobra-public-ip",
```

Comment:
i know you are doing this for your droplet testing, but we almost certainly won't be using this for actual deployments.  I don't have a request right now, but want you to keep that in mind.